docs/hacktoberfest/js/counter.js

```diff
@@ -0,0 +1,40 @@
+const renderTimer = () => {
+
+    const { distance, days, hours, minutes, seconds } = convertTime()
+
+    document.getElementById('days').innerHTML = parseValue(days)
+    document.getElementById('hours').innerHTML = parseValue(hours)
+    document.getElementById('minutes').innerHTML = parseValue(minutes)
+    document.getElementById('seconds').innerHTML = parseValue(seconds) 
+    
+    if (distance > 0){
+        setTimeout(renderTimer, 1000)
+    }
+}
+
+const parseValue = (value) => {    
+    return value < 10 ?  '0' + value  :  value
+} 
+
+const convertTime = () => {
+    const eventDate = 'Oct 19, 2019 09:00:00'
+
+    const countdownDate = new Date(eventDate).getTime()
+    const currentDate = new Date().getTime()
+    const distance  = countdownDate - currentDate
```

Comment:
né melhor intervalo o nome?
```suggestion
    const interval  = countdownDate - currentDate
```